packages/turbo/package.json

```diff
@@ -26,4 +26,4 @@
     "turbo-windows-64": "1.6.3",
     "turbo-windows-arm64": "1.6.3"
   }
-}
\ No newline at end of file
+}
```

Comment:
lol, this diff is why 8 other people got tagged in this PR 🤣 . Thanks Github, LGTM!